MODULE_ACCEPTANCE_CRITERIA.MD

```diff
@@ -44,7 +44,11 @@ Please see [Before Development](MODULE_EVALUATION_TEMPLATE#before-development) f
 * Module build MUST produce a valid module descriptor (3, 5)
   * _This is not applicable to libraries_
 * Module descriptor MUST include interface requirements for all consumed APIs (3, 5)
-  * _This is not applicable to libraries_
+  * For frontend modules and frontend libraries
+    * For each consumed API `package.json` MUST include the interface requirement in the `"okapiInterfaces"` or `"optionalOkapiInterfaces"` section
+      * -_note: read more at https://github.com/folio-org/stripes/blob/master/doc/dev-guide.md#the-package-file-stripes-entry_
+  * For backend modules
+    * For each consumed API the module descriptor MUST include the interface requirement in the `"requires"` or `"optional"` section
```

Comment:
We have separate sections for FE and BE, why not just move the separate criteria to those sections?

Reply:
I believe the intent of the criteria is general to both front-end and back-end modules. Only the mechanism of checking _may_ differ between the two (it doesn't have to). Thus my preference is to retain a single, general criteria, which could include guidance for how to check that is specific to different types